js/src/forum/components/SettingsPage.js

```diff
@@ -142,4 +151,19 @@ export default class SettingsPage extends UserPage {
 
     return items;
   }
+
+  /**
+   * Build an item list for the user's tokens controls.
+   *
+   * @return {ItemList}
+   */
+  tokensItems() {
```

Comment:
why not tokenItems?

Reply:
The other entry with a name was also plural (`notificationsItems`) so that's why I used plural as well. But I agree this sounds a bit odd.